docs/faq/help.txt

```diff
@@ -5,15 +5,35 @@ FAQ: Getting Help
 How do I do X? Why doesn't Y work? Where can I go to get help?
 ==============================================================
 
-If this FAQ doesn't contain an answer to your question, you might want to
-try the |django-users| mailing list. Feel free to ask any question related
-to installing, using, or debugging Django.
+First, please check if your question is answered on the
+:doc:`FAQ index <faq/index>`. Please also search for answers using your
```

Comment:
I would chop `index`, ``:doc:`FAQ <faq/index>` ``.

Reply:
Too many "please" IMO, maybe: _"Also, search ..."_.

Reply:
I'll adjust these and merge.